src/questionnaireitem_system_focus.js

```diff
@@ -0,0 +1,58 @@
+/**
+A QuestionnaireItemSystem that stores how often and how long the focus was moved away from the page.
+
+Reports as array in milliseconds how long the focus was lost each time.
```

Comment:
General question: what happens if a screen with this QuestionnaireItem is shown twice (e.g., pagination)?
From the implementation, I derived it would just append to the focusLostDuration array new entries.
Is this counted as a focus lost?

Reply:
When moving to a different page, releaseUI() is called and thus focus changes from now on are not considered. When returning to the site again, createUI() called again and all new events get appended.